components/theme/typography.ts

```diff
@@ -23,7 +23,11 @@
 import { css } from '@emotion/core';
 
 const baseFont = css`
-  font-family: Lato, sans-serif;
+  font-family: 'Roboto', sans-serif;
+`;
+
+const headingsFont = css`
+  font-family: 'Roboto', sans-serif;
```

Comment:
mew 2 fonts, as per spec

Reply:
just wondering the reason for a `headingsFont` that isn't different from `baseFont`?

Reply:
oooh copy pasta! nice catch